app/Http/Resources/CastResource.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class CastResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @param \Illuminate\Http\Request $request
+     * @return array|\Illuminate\Contracts\Support\Arrayable|\JsonSerializable
+     */
+    public function toArray($request)
+    {
+        //return parent::toArray($request);
+        return [
+            'id' => $this->id,
+            'medium_id' => $this->medium_id,
+            'person_id' => $this->person_id,
+            'gender_id' => $this->gender_id,
+            'character' => $this->character,
+            'created_at' => $this->created_at,
```

Comment:
change format returned date

Reply:
how should the date look ?

Reply:
and whether do it in model or just in resource?

Reply:
Maybe only date without time part
Please add this in resource

Reply:
See if there is a date anywhere on the mock-ups and adjust the format to it if you think it's ok